tx.spec

```diff
@@ -0,0 +1,36 @@
+# -*- mode: python -*-
+
+block_cipher = None
+
+
+a = Analysis(['tx'],
+             pathex=['C:\\code\\transifex\\transifex-client'],
```

Comment:
It's default file generated by PyInstaller. I haven't utilise block_cipher that much, but I think using block_cipher into some cryptography mode, will make the binary (*.pyc) get encrypted. https://github.com/pyinstaller/pyinstaller/issues/1408.

New code pushed